fe/fe-core/src/main/java/org/apache/doris/datasource/property/storage/AbstractS3CompatibleProperties.java

```diff
@@ -168,6 +169,10 @@ public AwsCredentialsProvider getAwsCredentialsProvider() {
                         sessionToken));
             }
         }
+        // For anonymous access (no credentials required)
+        if (StringUtils.isBlank(getAccessKey()) && StringUtils.isBlank(getSecretKey())) {
+            return AnonymousCredentialsProvider.create();
+        }
```

Comment:
This setup allows COS/OBS/OSS to be accessed without credentials. Have you tested whether they actually permit this kind of access?